docs/ScalarDLBackupCreationRestoration.md

```diff
@@ -0,0 +1,90 @@
+# Scalar DL Backup Creation on Kubernetes
+
+This guide shows you how to create and restore Scalar DL backups on Cosmos DB and DynamoDB without data inconsistencies.
+
+## Prerequisites
+
+* Read the Scalar DL backup creation and restoration guide, [see](https://github.com/scalar-labs/scalardl/blob/master/docs/backup-restore.md)
```

Comment:
Weird to add a link to `see`.

Reply:
I added `see` because the file name is `A Guide on How to Backup and Restore Data in Scalar DL` so I thought `see` is good.

I will update it with proper wordings